### DISCUSSION TITLE
Git repo is 338M but code base is 3.8M

### DISCUSSION THREAD

**Discussion by evrial** [gildas-lormeau/SingleFile#1433] [Q&A]
Any plans to purge large files from history to make reasonable size?

**Comment by gildas-lormeau** [comment]
AFAIK, it's impossible on GitHub.

**Comment by evrial** [comment]
https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository#purging-a-file-from-your-repositorys-history


> The git filter-repo tool and the BFG Repo-Cleaner rewrite your repository's history, which changes the SHAs for existing commits that you alter and any dependent commits. Changed commit SHAs may affect open pull requests in your repository. We recommend merging or closing all open pull requests before removing files from your repository.
